hieradata_aws/production.yaml

```diff
@@ -258,3 +258,843 @@ govuk_datascrubber::ensure: 'latest'
+    host_aliases:
+      - whitehall-frontend-3.frontend
+      - whitehall-frontend-3
+  licensify-lb-1.licensify.publishing.service.gov.uk:
+    ip: 10.5.0.101
+    host_aliases:
+      - licensify-lb-1.licensify
+      - licensify-lb-1
+  licensify-internal-lb.licensify.publishing.service.gov.uk:
+    ip: 10.5.1.2
+    host_aliases:
+      - licensify-internal-lb.licensify
+      - licensify-internal-lb
+  licensify-lb-2.licensify.publishing.service.gov.uk:
+    ip: 10.5.0.102
+    host_aliases:
+      - licensify-lb-2.licensify
+      - licensify-lb-2
+  # bouncer-2.redirector.publishing.service.gov.uk:
+  #   ip: 10.3.5.5
+  #   host_aliases:
+  #     - bouncer-2.redirector
+  #     - bouncer-2
+  # bouncer-1.redirector.publishing.service.gov.uk:
+  #   ip: 10.3.5.4
+  #   host_aliases:
+  #     - bouncer-1.redirector
+  #     - bouncer-1
+  # bouncer-3.redirector.publishing.service.gov.uk:
+  #   ip: 10.3.5.6
+  #   host_aliases:
+  #     - bouncer-3.redirector
+  #     - bouncer-3
+  cache-1.router.publishing.service.gov.uk:
+    ip: 10.3.1.2
+    host_aliases:
+      - cache-1.router
+      - cache-1
+  router-backend-2.router.publishing.service.gov.uk:
+    ip: 10.3.1.11
+    host_aliases:
+      - router-backend-2.router
+      - router-backend-2
+  # jumpbox-1.management.publishing.service.gov.uk:
+  #   ip: 10.3.0.100
+  #   host_aliases:
+  #     - jumpbox-1.management
+  #     - jumpbox-1
+  # elasticsearch-2.backend.publishing.service.gov.uk:
+  #   ip: 10.3.3.16
+  #   host_aliases:
+  #     - elasticsearch-2.backend
+  #     - elasticsearch-2
+  whitehall-frontend-2.frontend.publishing.service.gov.uk:
+    ip: 10.3.2.6
+    host_aliases:
+      - whitehall-frontend-2.frontend
+      - whitehall-frontend-2
+  # bouncer-vse-lb.redirector.publishing.service.gov.uk:
+  #   ip: 37.26.90.219
+  #   host_aliases:
+  #     - bouncer-vse-lb.redirector
+  #     - bouncer-vse-lb
+  # elasticsearch-1.backend.publishing.service.gov.uk:
+  #   ip: 10.3.3.15
+  #   host_aliases:
+  #     - elasticsearch-1.backend
+  #     - elasticsearch-1
+  calculators-frontend-2.frontend.publishing.service.gov.uk:
+    ip: 10.3.2.12
+    host_aliases:
+      - calculators-frontend-2.frontend
+      - calculators-frontend-2
+  draft-cache-2.router.publishing.service.gov.uk:
+    ip: 10.3.1.201
+    host_aliases:
+      - draft-cache-2.router
+      - draft-cache-2
+  # mongo-2.backend.publishing.service.gov.uk:
+  #   ip: 10.3.3.7
+  #   host_aliases:
+  #     - mongo-2.backend
+  #     - mongo-2
+  # mongo-1.backend.publishing.service.gov.uk:
+  #   ip: 10.3.3.6
+  #   host_aliases:
+  #     - mongo-1.backend
+  #     - mongodb.cluster
+  #     - mongo-1
+  backend-internal-lb.backend.publishing.service.gov.uk:
+    ip: 10.3.3.254
+    host_aliases:
+      - backend-internal-lb.backend
+      - backend-internal-lb
+      - asset-manager.publishing.service.gov.uk
+      - canary-backend.publishing.service.gov.uk
+      - ckan.publishing.service.gov.uk
```

Comment:
CKAN has been flagged by PH as "migrated" (though still in development) I think it is safest to remove from here